### DISCUSSION TITLE
How to run stockfish chess engine on raspberry pi???

### DISCUSSION THREAD

**Discussion by Tavs44** [community/community#118976] [Programming Help] [answered]
### Select Topic Area

Question

### Body

I'm currently using raspberry pi 4. I'm looking for a way to run stockfish 16 chess engine.

**Comment by 0YJ** [comment, marked as answer]
you should not have pi lite OS since it's command line based, use graphic OS and open termial, run ```sudo apt update``` then ```sudo apt install stockfish``` should give u the game i think.

**Comment by LiteBrite82** [comment]
Thanks for posting in the GitHub Community, @Tavs44 !

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Discussions category is solely related to conversations around the [GitHub product Discussions](https://docs.github.com/en/discussions/collaborating-with-your-community-using-discussions/about-discussions). This question should be in the Programming Help or General category. I've gone ahead and moved it for you. Good luck!
